package/feature/feature-nest-stock/src/stock.service.ts

```diff
@@ -265,6 +265,106 @@ export class StockService {
     return result;
   }
 
+  async drawStockInfo(stockId: string, body: Request.PostDrawStockInfo): Promise<Stock> {
+    const { userId } = body;
+    console.debug('drawStockInfo', { userId });
+    let result: Stock;
+
+    const session = await this.connection.startSession();
+    await session.withTransaction(async () => {
+      const stock = await this.stockRepository.findOneById(stockId, undefined, { session });
+      const user = await this.userRepository.findOne({ stockId, userId }, undefined, { session });
+
+      if (!stock.isTransaction) {
+        throw new HttpException('지금은 거래할 수 없습니다', HttpStatus.CONFLICT);
+      }
+
+      if (!user) {
+        throw new HttpException('유저 정보를 불러올 수 없습니다', HttpStatus.CONFLICT);
+      }
+
+      // 현재 라운드에 해당하는 시점의 idx
+      const timeIdx = Math.min(
+        Math.floor(getDateDistance(stock.startedTime, new Date()).minutes / stock.fluctuationsInterval),
+        9,
+      );
+
+      const nextTimeIdx = timeIdx + 1;
```

Comment:
```suggestion
      const nextTimeIdx = timeIdx + 2;
```

다음 라운드가 1초 남았는데 정보를 뽑는 불상사가 있기 때문에
위와 같이 해야하지 않을까 합니다.

그리고 8라운드부터는 해당 API가 작동하지 않도록 해야할듯 해요.

Reply:
네네 다음 라운드의 주식 정보는 포함하지 않는걸로 하죠